modules/end-user-guide/partials/assembly_creating-a-workspace-from-included-options.adoc

```diff
@@ -0,0 +1,20 @@
+:parent-context-of-creating-a-workspace-from-included-options: {context}
+
+[id="creating-a-workspace-from-included-options_{context}"]
+= Creating a workspace from included options
+
+:context: creating-a-workspace-from-included-options
+
+{prod-short} includes optional workspace samples and devfile templates to simplify workspace creation.
+
+. xref:creating-a-workspace-from-workspace-sample_{context}[Creating a workspace from workspace sample]
```

Comment:
use * to create a bullet list of possible options. Numbered lists are for procedure steps :)